es-app/src/guis/GuiInfoPopup.h

```diff
@@ -14,7 +14,7 @@ class GuiInfoPopup : public GuiComponent, public Window::InfoPopup
 	GuiInfoPopup(Window* window, std::string message, int duration, int fadein = 500, int fadeout = 500);
 	~GuiInfoPopup();
 	void render(const Transform4x4f& parentTrans) override;
-	inline void stop() { running = false; };
```

Comment:
In file included from /home/devel/temp/EmulationStation_Stable/es-app/src/guis/GuiMenu.cpp:24:       
/home/devel/temp/EmulationStation_Stable/es-app/src/guis/GuiInfoPopup.h:17:14: warning: 'stop' overri
des a member function but is not marked 'override' [-Winconsistent-missing-override]
        inline void stop() { running = false; };
                    ^
/home/devel/temp/EmulationStation_Stable/es-core/src/Window.h:40:16: note: overridden virtual functio
n is here
                virtual void stop() = 0;
                             ^